pkg/util/log/eventpb/telemetry.proto

```diff
@@ -71,3 +71,32 @@ message CapturedIndexUsageStats {
   bool is_unique = 10 [(gogoproto.jsontag) = ",omitempty"];
   bool is_inverted = 11 [(gogoproto.jsontag) = ",omitempty"];
 }
+
+// CreateChangefeed is an event for any CREATE CHANGEFEED query that
+// successfully starts running.  Failed CREATE statements will show up as
+// ChangefeedFailed events.
+message CreateChangefeed {
+  CommonChangefeedEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+
+  // The number of tables listed in the query that the changefeed is to run on.
+  int32 num_tables = 2 [(gogoproto.jsontag) = ",omitempty"];
+
+  // The behavior of emitted resolved spans (ex: yes, no, 10s)
+  string resolved = 3 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) =  "redact:\"nonsensitive\""];
```

Comment:
I kinda think that just indication if resolved option was used or not is probably enough.
But okay..